README.md

```diff
@@ -27,6 +27,19 @@ CREATE TABLE paypal (
   currency TEXT,
   country_code TEXT
 );
+
+CREATE TABLE stripe (
+  id TEXT PRIMARY KEY,
+  timestamp TIMESTAMP WITH TIME ZONE,
+  amount INTEGER,
```

Comment:
I don't think it should be an integer though. What if I donate $10.10 USD? We'd lose the $0.10 precision